src/App.tsx

```diff
@@ -1,29 +1,12 @@
-import { useState } from "react";
-import reactLogo from "./assets/react.svg";
-import viteLogo from "/vite.svg";
+
 import "./App.css";
+import HomePage from './pages/HomePage/HomePage';
```

Comment:
Inconsistent single/double quote. Remember install Prettier extension and turn on "Format on Save" in editor.